pinot-query-runtime/src/test/java/org/apache/pinot/query/QueryServerEnclosure.java

```diff
@@ -112,7 +112,8 @@ public void shutDown() {
 
   public CompletableFuture<Void> processQuery(WorkerMetadata workerMetadata, StagePlan stagePlan,
       Map<String, String> requestMetadataMap) {
-    return CompletableFuture.runAsync(() -> _queryRunner.processQuery(workerMetadata, stagePlan, requestMetadataMap),
+    return CompletableFuture.runAsync(
+        () -> _queryRunner.processQuery(workerMetadata, stagePlan, requestMetadataMap, null),
```

Comment:
QueryRunner.processQuery last argument is not annotated with Nullable so we should not call it with null. We need to either change the QueryRunner method signature or provide a value here.

Reply:
Done